packages/@sanity/form-builder/src/inputs/common/conditionalReadOnly/ConditionalReadOnlyContext.ts

```diff
@@ -0,0 +1,9 @@
+import {createContext} from 'react'
```

Comment:
How does this relate to the ConditionalReadOnlyContext exported from @sanity/base? Do we still need both?